### PR TITLE
[EV-1873] Add volume mount with the TLS key pair to queryserver conta…

### PR DESCRIPTION
**Pull request by dimitri-nicolo**
…iner def.

## Description

The PR aims to add a new volume mount to the queryserver container within the apiserver deployment. The volume mount contains the path to the tls key pair for both standalone and management clusters.

## For PR author

- [ ] Tests for change.
- [ ] If changing pkg/apis/, run `make gen-files`
- [ ] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [ ] Milestone set according to targeted release.
- [ ] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

**Review by doublek — APPROVED** [review]
LGTM

File: pkg/render/apiserver.go [hunk 1]
```diff
@@ -918,6 +918,15 @@ func (c *apiServerComponent) queryServerContainer() corev1.Container {
 		env = append(env, corev1.EnvVar{Name: "MULTI_INTERFACE_MODE", Value: c.cfg.Installation.CalicoNetwork.MultiInterfaceMode.Value()})
 	}
 
+	volumeMounts := []corev1.VolumeMount{
+		c.cfg.TLSKeyPair.VolumeMount(c.SupportedOSType()),
+	}
+	if c.cfg.Installation.Variant == operatorv1.TigeraSecureEnterprise {
```
[the remark this answers is not in the document]
Comment (dimitri-nicolo) [reply pkg/render/apiserver.go]: @rene-dekker I've adressed your comment and removed the additional volume mount. Also, have added the new tigera-queryserver container volumeMount expected results to the apiserver_test.go

**Comment by rene-dekker** [comment]
Now that queryserver has authentication, do we also need tokenreviews api permissions?

**Review by rene-dekker — APPROVED** [review]
lgtm

**Pull request merged**